src/modules/customers/customers.service.test.ts

```diff
@@ -33,4 +48,17 @@ describe('CustomerService', () => {
       expect(response).toEqual(getCustomersResponse[0]);
     });
   });
+
+  describe('getDunAndBradstreetNumberByRegistrationNumber', () => {
+    const companyRegistrationNumber = '12341234';
+    const dunsNumber = '56785678';
+
+    it('returns duns number for the registration number', async () => {
```

Comment:
Can we add an assertion for following cases please:

1. Does not returns DNB number.
2. Company registration number being empty or invalid i.e. `TEST`.